components/TransactionPaginated.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="container">
```

Comment:
![image](https://user-images.githubusercontent.com/72648845/186087077-c5078de1-a05c-48cd-9e64-f2fc1e3880c9.png)
strange component UI work. Fix this please